k2/csrc/context.h

```diff
@@ -468,7 +476,7 @@ class ParallelRunnerDummy {
   cudaStream_t stream_;
 };
 
-using ParallelRunner = ParallelRunnerDummy;
+using ParallelRunner = ParallelRunnerActive;
```

Comment:
Why is it switched back? `NewStream()` always returns the default stream in the current code.

Reply:
As the comments showed above, we decided that if `num_work_items < a threshould, e.g. 10k` we don't create a new stream (as Dan found it would be heavy if we create new stream every time (recalling the event stuff related to this), so he got an idea that we only create new stream when >10k)

Reply:
The point is that the current code always uses the default argument 0.

Reply:
Yes, so it's now still equivalent to ParallelRunnerDummy, as you asked in another quesiton.